packages/l/ltrace/package.yml

```diff
@@ -1,14 +1,13 @@
 name       : ltrace
 version    : 0.7.3
-release    : 4
+release    : 5
 source     :
     - http://www.ltrace.org/ltrace_0.7.3.orig.tar.bz2 : 0e6f8c077471b544c06def7192d983861ad2f8688dd5504beae62f0c5f5b9503
+homepage   : https://www.ltrace.org/
 license    : GPL-2.0
 component  : programming.tools
 summary    : Tracks runtime library calls in dynamically linked programs
-description: "ltrace intercepts and records dynamic library calls which are called\
-    \ by an executed process and the signals received by that process. It can also\
-    \ intercept and print the system calls executed by the program. \n"
+description: ltrace intercepts and records dynamic library calls which are called by an executed process and the signals received by that process. It can also intercept and print the system calls executed by the program.
```

Comment:
typically the description will be like:

```yaml
description: |
    blahblahblah
```

can you fix it to match other packages?